depthai_examples/launch/stereo_inertial_node.launch.py

```diff
@@ -21,7 +21,7 @@ def generate_launch_description():
                                 'rviz', 'stereoInertial.rviz')
     default_resources_path = os.path.join(depthai_examples_path,
                                 'resources')
-
+    ipAddress    = LaunchConfiguration('ipAddress',     default = 'x')
     mxId         = LaunchConfiguration('mxId',      default = 'x')
```

Comment:
it is `x` here. you are checking for empty in the code.

Reply:
@saching13  Good catch! I fixed it with the new commit